docs/changes.rst

```diff
@@ -37,6 +37,9 @@ Released: not yet
   ``pywbem.configure_logging()`` function that disables logging.
   (part of issue #86)
 
+* Reduced memory consumption of CIM objects and CIM types by using slots.
```

Comment:
how about by replacing existing attributes with python slots.

Reply:
They are still attributes. I changed it to "... by defining their attributes to use Python slots."